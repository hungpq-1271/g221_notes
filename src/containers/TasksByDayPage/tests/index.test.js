import React from 'react'
import { render } from '@testing-library/react'

import TasksByDay from '../index'

describe('<TasksByDay />', () => {
  it('should render and match the snapshot', () => {
    const { container } = render(<TasksByDay />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
