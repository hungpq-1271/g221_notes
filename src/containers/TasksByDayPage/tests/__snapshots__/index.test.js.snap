// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TasksByDay /> should render and match the snapshot 1`] = `
<div
  class="MuiContainer-root MuiContainer-maxWidthMd"
>
  <div
    style="margin-top: 30px;"
  >
    <div
      style="display: flex; flex-direction: row-reverse;"
    >
      <button
        aria-label="add"
        class="MuiButtonBase-root MuiFab-root makeStyles-margin-10 MuiFab-extended"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiFab-label"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
          Add
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <button
        aria-label="filter"
        class="MuiButtonBase-root MuiFab-root makeStyles-margin-10 MuiFab-extended"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiFab-label"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"
            />
          </svg>
          Filter
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
    <div
      style="display: flex; margin-top: 30px;"
    >
      <div
        class="MuiTabs-root"
      >
        <div
          class="MuiTabs-flexContainer WithStyles(ForwardRef(Tabs))-flexContainer-37"
        >
          <div
            class="MuiTabs-scroller MuiTabs-fixed"
            role="tablist"
            style="overflow: hidden;"
          >
            <div
              class="MuiTabs-flexContainer WithStyles(ForwardRef(Tabs))-flexContainer-37"
            >
              <button
                aria-selected="true"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit Mui-selected WithStyles(ForwardRef(Tab))-selected-48"
                role="tab"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  ALL
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit"
                role="tab"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  DONE
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit"
                role="tab"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  NOT DONE
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
            <span
              class="PrivateTabIndicator-root-59 PrivateTabIndicator-colorSecondary-61 MuiTabs-indicator WithStyles(ForwardRef(Tabs))-indicator-38"
              style="left: 0px; width: 0px;"
            />
          </div>
        </div>
      </div>
      <div
        class="MuiTypography-root MuiTypography-body1"
        style="padding: 0px 24px 24px 24px; width: 80%;"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-8 MuiPaper-rounded"
        >
          <div
            class="MuiCardContent-root"
          >
            <p
              class="MuiTypography-root makeStyles-title-9 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              Word of the Day
            </p>
            <p
              class="MuiTypography-root MuiTypography-body2"
            >
              well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.
            </p>
          </div>
          <div
            style="display: flex; flex-direction: row-reverse;"
          >
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-button-11 MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-leftIcon-12"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M9 16.2L4.8 12l-1.4 1.4L9 19 21 7l-1.4-1.4L9 16.2z"
                  />
                </svg>
                Done
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-button-11 MuiButton-contained MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-leftIcon-12"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                  />
                </svg>
                Edit
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-button-11 MuiButton-contained MuiButton-containedSecondary MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-leftIcon-12"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
                  />
                </svg>
                Delete
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-8 MuiPaper-rounded"
        >
          <div
            class="MuiCardContent-root"
          >
            <p
              class="MuiTypography-root makeStyles-title-9 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              Word of the Day
            </p>
            <p
              class="MuiTypography-root MuiTypography-body2"
            >
              well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.
            </p>
          </div>
          <div
            style="display: flex; flex-direction: row-reverse;"
          >
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-button-11 MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-leftIcon-12"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M9 16.2L4.8 12l-1.4 1.4L9 19 21 7l-1.4-1.4L9 16.2z"
                  />
                </svg>
                Done
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-button-11 MuiButton-contained MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-leftIcon-12"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                  />
                </svg>
                Edit
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-button-11 MuiButton-contained MuiButton-containedSecondary MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-leftIcon-12"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
                  />
                </svg>
                Delete
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-8 MuiPaper-rounded"
        >
          <div
            class="MuiCardContent-root"
          >
            <p
              class="MuiTypography-root makeStyles-title-9 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              Word of the Day
            </p>
            <p
              class="MuiTypography-root MuiTypography-body2"
            >
              well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.
            </p>
          </div>
          <div
            style="display: flex; flex-direction: row-reverse;"
          >
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-button-11 MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-leftIcon-12"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M9 16.2L4.8 12l-1.4 1.4L9 19 21 7l-1.4-1.4L9 16.2z"
                  />
                </svg>
                Done
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-button-11 MuiButton-contained MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-leftIcon-12"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                  />
                </svg>
                Edit
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-button-11 MuiButton-contained MuiButton-containedSecondary MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-leftIcon-12"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
                  />
                </svg>
                Delete
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-8 MuiPaper-rounded"
        >
          <div
            class="MuiCardContent-root"
          >
            <p
              class="MuiTypography-root makeStyles-title-9 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              Word of the Day
            </p>
            <p
              class="MuiTypography-root MuiTypography-body2"
            >
              well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.well meaning and kindly. well meaning and kindly.
            </p>
          </div>
          <div
            style="display: flex; flex-direction: row-reverse;"
          >
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-button-11 MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-leftIcon-12"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M9 16.2L4.8 12l-1.4 1.4L9 19 21 7l-1.4-1.4L9 16.2z"
                  />
                </svg>
                Done
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-button-11 MuiButton-contained MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-leftIcon-12"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                  />
                </svg>
                Edit
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-button-11 MuiButton-contained MuiButton-containedSecondary MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-leftIcon-12"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
                  />
                </svg>
                Delete
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
